lib/web_ui/dev/browser_lock.yaml

```diff
@@ -13,10 +13,11 @@ chrome:
   # `self.m.platform.name.capitalize()` evaluates to. See:
   #
   # recipe_modules/web_util/api.py
-  Linux: 929514
-  Mac: 929514
-  Win: 929514
-  version: '96.2' # CIPD tag for the above Build IDs. Normally "ChromeMajorVersion.UploadAttempt". ;)
+  Linux: 1046597
+  Mac: 1046592
+  Mac_Arm: 1046589
+  Win: 1046600
+  version: '107.0' # CIPD tag for the above Build IDs. Normally "ChromeMajorVersion.UploadAttempt". ;)
```

Comment:
What do you think of introducing a new variant (can be Linux-only for now) for a more recent Chromium? Similar to `Mac`/`Mac_Arm` we could have `Linux`/`Linux_Canary`. We wanted to be able to run on stable the upcoming versions of Chromium for a while now so we could alert the Chromium project about any incoming regressions. That should meet the needs for the WasmGC experiment, or any future features we're experimenting with.

Reply:
That won't work with this current setup, because the version number is the same for all platforms. In general, I feel like the definition of one "platform" being ahead version-wise is confusing and counter-intuitive. I think having support for running on non-stable versions is probably a little out of scope for this particular change. I've made a change to push us back to 105.0, which is the stable version of Chrome. I might augment this a bit more for the wasm changes in a subsequent PR, if 107 is still too new by the time I want to merge the wasm harness stuff.